generators/vsts/templates/asp_build.json

```diff
@@ -28,18 +28,33 @@
          "enabled": true,
          "continueOnError": false,
          "alwaysRun": false,
-         "displayName": "Run dotnet restore",
+         "displayName": "Restore",
          "timeoutInMinutes": 0,
          "task": {
-            "id": "d9bafed4-0b18-4f58-968d-86655b4d2ce9",
+            "id": "5541a522-603c-47ad-91fc-a4b1d163081b",
             "versionSpec": "*",
             "definitionType": "task"
          },
          "inputs": {
-            "filename": "dotnet",
-            "arguments": "restore --no-cache",
```

Comment:
The --no-cache argument is important for hosted builds https://docs.microsoft.com/en-us/dotnet/articles/core/tools/dotnet-restore and should be still be passed as an argument.